virttest/qemu_vm.py

```diff
@@ -2056,9 +2046,12 @@ def sort_key(dev):
         elif params.get("display") == "nographic":
             cmd += add_nographic()
         elif params.get("display") == "spice":
+            spice_options = {}
             if params.get("rhel5_spice"):
                 spice_params = params.get("spice_params")
-                cmd += add_spice_rhel5(devices, spice_params)
+                tmp_cmd, spice_options["spice_port"] =\
```

Comment:
In style guide we encourage the use of brackets to escaped `\n`s. How about `ret = add_spice_rhel5(...); tmp_cmd, spice_options... = ret`?